CHANGELOG.rst

```diff
@@ -1,5 +1,22 @@
 Changelog
 =========
+0.3.0
+`````
+
+Version 0.3.0 updates python-bsonjs's vendored copy of libbson to 1.20.0
+http://mongoc.org/libbson/1.20.0/
+https://github.com/mongodb/mongo-c-driver/blob/1.20.0/src/libbson/NEWS
+
+This release also provides a number of improvements and features:
+
+- Added testing and support for Windows
```

Comment:
Windows was already tested and supported.

Reply:
Done.